packages/client/src/execution/vmexecution.ts

```diff
@@ -528,7 +528,7 @@ export class VMExecution extends Execution {
                 this.config.execCommon.gteHardfork(Hardfork.Paris) === true
                   ? ''
                   : `td=${this.chain.blocks.td} `
-              this.config.logger.info(
+              this.config.logger.verbose(
```

Comment:
What is this change doing? 🤔 I am not even finding information about this log level (when googling)?

Reply:
Ah, just testing myself and seeing that this seems to really removing the log message for default setup. So this is one of the most central logging output I am using all the time.

I would urgently want to get this in again, otherwise it is not really possible anymore to follow on a "classic sync" (mainnet), especially since import and execution deviates so much for our client.

Will re-introduce along some other follow-up client log work.

Reply:
(so for pre-Merge this is definitely necessary)

Reply:
Another update: ok, but going a bit back here, my client just switched over to now doing the execution and for this situation I also definitely want to see what is going on on the execution side:

![grafik](https://github.com/ethereumjs/ethereumjs-monorepo/assets/931137/203a0a45-4f17-4aa7-8bcc-6fe5d0f55952)

So I would rather want to re-introduce, at least for most contexts.